yaptide/static/openapi.yaml

```diff
@@ -410,7 +411,7 @@ paths:
                 $ref: '#/components/schemas/BasicResponse'
   /jobs/direct:
     post:
-      summary: Allows for submitting jobs
+      summary: Allows for submitting jobs to run directly on server.
```

Comment:
which server ? `ares.cyfronet.pl` ?
be more specific that we mean `dedicated machine in cloud`.
Mention that here the job doesn't wait in the queue as with SLURM.